Java/src/test/java/com/gildedrose/TexttestFixture.java

```diff
@@ -1,5 +1,7 @@
 package com.gildedrose;
 
+import com.gildedrose.item.Item;
+
 public class TexttestFixture {
```

Comment:
What is this class?

Reply:
I don't know, it was already there! I'm not using it anywhere...